libs/@local/hashql/core/src/module/std_lib/core/graph/types/principal/actor_group/mod.rs

```diff
@@ -0,0 +1,43 @@
+pub(in crate::module::std_lib) mod web;
+
+use crate::{
+    heap::Heap,
+    module::{
+        StandardLibrary,
+        std_lib::{self, ItemDef, ModuleDef, StandardLibraryModule},
+    },
+    symbol::Symbol,
+};
+
+pub(in crate::module::std_lib) struct ActorGroup {
+    _dependencies: (std_lib::core::uuid::Uuid,),
+}
+
+impl<'heap> StandardLibraryModule<'heap> for ActorGroup {
+    type Children = (self::web::Web,);
+
+    fn name(heap: &'heap Heap) -> Symbol<'heap> {
+        heap.intern_symbol("actor_group")
+    }
+
+    fn define(lib: &mut StandardLibrary<'_, 'heap>) -> ModuleDef<'heap> {
+        let mut def = ModuleDef::new();
+        let heap = lib.heap;
+
+        // newtype ActorGroupEntityUuid = EntityUuid;
+        // (we just set it to Uuid to avoid any cycles)
```

Comment:
Could you add `TODO` or something to make it more visible? A linear ticket ID or link would be great (H-4616?)